homeassistant/components/analytics/const.py

```diff
@@ -0,0 +1,88 @@
+"""Constants for the analytics integration."""
+from datetime import timedelta
+from enum import Enum
+import logging
+
+ANALYTICS_ENDPOINT_URL = "https://floral-resonance-f63b.ludeeus.workers.dev"
```

Comment:
Will this url change before merge?

Reply:
yes, it will be `https://analytics-api.home-assistant.io`, but that is not ready so I used my own namespace while testing.